determined-ai/determined
# chore: change min npm version

## Description
Follow up of #5926




## Test Plan
- Run `make clean` to remove `build`, `node_modules` etc
- Run `npm i` or `make get-deps` to install npm modules
- Run `make live` to check if it works
- Run `make build` to check if module can be built
- Check if CI pass




## Commentary (optional)
Adjusted `package-lock.json` for npm v9. Lock file version is now v3. 
npm v9 is compatible with our current node version ^v16.13. [resource](https://github.blog/changelog/2022-10-24-npm-v9-0-0-released/#%E2%9A%A0%EF%B8%8F-notable-breaking-changes)




## Checklist

  See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.

## Ticket
